docker/nginx/conf.d/include/proxy-cache-downloads

```diff
@@ -1,21 +1,12 @@
-proxy_cache skynet; # cache name
-proxy_cache_key $skylink_v1$path$arg_format$arg_attachment$arg_start$arg_end$http_range; # unique cache key
-proxy_cache_min_uses 3; # cache after 3 uses
-proxy_cache_valid 200 206 307 308 48h; # keep 200, 206, 307 and 308 responses valid for up to 2 days
 add_header X-Proxy-Cache $upstream_cache_status; # add response header to indicate cache hits and misses
 
-# map skyd env variable value to "1" for true and "0" for false (expected by proxy_no_cache)
-set_by_lua_block $skyd_disk_cache_enabled {
-    return os.getenv("SKYD_DISK_CACHE_ENABLED") == "true" and "1" or "0"
-}
-
 # bypass - this will bypass cache hit on request (status BYPASS)
 # but still stores file in cache if cache conditions are met
-proxy_cache_bypass $cookie_nocache $arg_nocache $skyd_disk_cache_enabled;
+proxy_cache_bypass $cookie_nocache $arg_nocache "1";
 
 # no cache - this will ignore cache on request (status MISS)
 # and does not store file in cache under no condition
-set_if_empty $nocache "0";
+set $nocache "1";
 
 # disable cache when nocache is set or skyd cache is enabled
-proxy_no_cache $nocache $skyd_disk_cache_enabled;
+proxy_no_cache $nocache;
```

Comment:
do you still need a variable here? isn't `$nocache` always "1" here, I think we removed all the variables here so we can probably get rid of the `$nocache` var and just set `proxy_no_cache` directly

Reply:
Probably not. I was afraid to touch more than I had to but I see I might need to.